### PR TITLE
More examples

### PR DESCRIPTION
**Pull request by joeduffy**
Ingest the latest pulumi/pulumi-terraform, bringing a lot more
examples to our documentation.

### CODE REVIEW

File: sdk/go/aws/pinpoint/smsChannel.go [hunk 1]
```diff
@@ -8,6 +8,34 @@ import (
 	"github.com/pulumi/pulumi/sdk/go/pulumi"
 )
 
+// ## ---
```
Comment (joeduffy) [inline sdk/go/aws/pinpoint/smsChannel.go]: This is clearly wrong.

File: sdk/go/aws/rds/eventSubscription.go [hunk 2]
```diff
@@ -9,6 +9,14 @@ import (
 )
 
 // Provides a DB event subscription resource.
+// 
+// ## Attributes
```
Comment (joeduffy) [inline sdk/go/aws/rds/eventSubscription.go]: Not sure why this isn't parsed correctly.

Comment (joeduffy) [reply sdk/go/aws/rds/eventSubscription.go]: Oh, I see, `Attributes` was never something we looked for. Will fix.

File: sdk/nodejs/acm/certificateValidation.ts [hunk 3]
```diff
@@ -13,110 +13,6 @@ import * as utilities from "../utilities";
  * deploy the required validation records and wait for validation to complete.
  * 
  * > **WARNING:** This resource implements a part of the validation workflow. It does not represent a real-world entity in AWS, therefore changing or deleting this resource on its own has no immediate effect.
- * 
- * 
- * ## Example Usage
```
Comment (joeduffy) [inline sdk/nodejs/acm/certificateValidation.ts]: Not sure why these examples disappeared.

**Comment by jen20** [comment]
I _think_ this should be obviated by #462 which pulls in the latest `pulumi-terraform` also? Sorry I didn't get to this one sooner!

**Pull request closed without merging**